SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -6456,7 +6456,7 @@ void CEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container,
   unsigned short iDim;
   unsigned long iVertex, iPoint;
   su2double P_Total, T_Total, Velocity[MAXNDIM], Velocity2, H_Total, Temperature, Riemann,
-  Pressure, Density, Energy, *Flow_Dir, Mach2, SoundSpeed2, SoundSpeed_Total2, Vel_Mag,
+  Pressure, Density, Energy, Flow_Dir[MAXNDIM] = {0.0}, Mach2, SoundSpeed2, SoundSpeed_Total2, Vel_Mag,
```

Comment:
Nice catch. But why initialize Flow_Dir and not Velocity (same for UnitNormal and Normal)?

Reply:
Sometimes the compiler will complain that some of the values in the array may be used uninitialised. For example if we mix loops over nDim with loops over MAXNDIM.